### PR TITLE
TINKERPOP-1369 Renamed REST API to HTTP API

### PR DESCRIPTION
**Pull request by robertdale**

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -59,6 +59,7 @@ TinkerPop 3.2.2 (NOT OFFICIALLY RELEASED YET)
 * Added color preferences
 * Added input, result prompt preferences
 * Added multi-line indicator in Gremlin Console
+* Replaced term `REST` with `HTTP` to remove any confusion as to the design of the API
```
Comment (spmallette) [inline CHANGELOG.asciidoc]: when 3.3.0 CHANGELOG is setup, this entry will need to move there prior to merge

**Comment by okram** [comment]
I think is smart. This way all those REST evangelists won't get all uppity.

Its funny how REST people are like Ruby/Clojure/Hadoop people -- "bro, insider tip --- its all about REST. Solves all your problems."

VOTE +1

**Comment by PommeVerte** [comment]
+1 from me :)

**Comment by dkuppitz** [comment]
VOTE: +1

**Comment by spmallette** [comment]
I will merge this once 3.3.x is opened up for development.

**Comment by spmallette** [comment]
this has been merged to master - note that the hash got re-written to 43f10d53307cfc4ec3dcaad208e57f403624b67e because i had to rebase at some point.  could you please close this on your end @robertdale

**Comment by robertdale** [comment]
okie dokie

**Pull request closed without merging**